runtime/reference/ts_config_migration.md

```diff
@@ -0,0 +1,501 @@
+can be a fully qualified URL as well:
+
+```ts
+/// <reference types="https://deno.land/x/pkg@1.0.0/types.d.ts" />
+```
+
+### Suppling "types" in deno.json
+
+Another option is to provide a `"types"` value to the `"compilerOptions"` in
+your `deno.json`. For example:
+
+```json title="deno.json"
+{
+  "compilerOptions": {
+    "types": [
+      "./types.d.ts",
+      "https://deno.land/x/pkg@1.0.0/types.d.ts",
+      "/Users/me/pkg/types.d.ts"
+    ]
+  }
+}
+```
+
+Like the triple-slash reference above, the specifier supplied in the `"types"`
+array will be resolved like other specifiers in Deno. In the case of relative
+specifiers, it will be resolved relative to the path to the config file. Make
+sure to tell Deno to use this file by specifying `--config=path/to/file` flag.
+
+## Type Checking Web Workers
+
+When Deno loads a TypeScript module in a web worker, it will automatically type
+check the module and its dependencies against the Deno web worker library. This
+can present a challenge in other contexts like `deno cache` or in editors. There
+are a couple of ways to instruct Deno to use the worker libraries instead of the
+standard Deno libraries.
+
+### Triple-slash directives
+
+This option couples the library settings with the code itself. By adding the
+following triple-slash directives near the top of the entry point file for the
+worker script, Deno will now type check it as a Deno worker script, irrespective
+of how the module is analyzed:
+
+```ts
+/// <reference no-default-lib="true" />
+/// <reference lib="deno.worker" />
+```
+
+The first directive ensures that no other default libraries are used. If this is
+omitted, you will get some conflicting type definitions, because Deno will try
+to apply the standard Deno library as well. The second instructs Deno to apply
+the built-in Deno worker type definitions plus dependent libraries (like
+`"esnext"`).
+
+When you run a `deno cache` or `deno bundle` command or use an IDE which uses
+the Deno language server, Deno should automatically detect these directives and
+apply the correct libraries when type checking.
+
+The one disadvantage of this, is that it makes the code less portable to other
+non-Deno platforms like `tsc`, as it is only Deno which has the `"deno.worker"`
+library built into it.
+
+### Providing "lib" setting in deno.json
+
+You can provide a "lib" option in your `deno.json` file to instruct Deno to use
+library files. For example:
+
+```json title="deno.json"
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "lib": ["deno.worker"]
+  }
+}
+```
+
+Then when running deno subcommand, you would need to pass the
+`--config path/to/file` argument, or if you are using an IDE which leverages the
+Deno language server, set the `deno.config` setting.
+
+If you also have non-worker scripts, you will either need to omit the `--config`
+argument, or have one that is configured to meet the needs of your non-worker
+scripts.
```

Comment:
This seems to duplicate an earlier section about supporting the browser.

Reply:
I wasn't sure whether it was worth reiterating that it works for workers too, think we should take it out?